doc/source/whatsnew/v0.24.0.txt

```diff
@@ -185,7 +185,7 @@ Other Enhancements
 - :class:`Resampler` now is iterable like :class:`GroupBy` (:issue:`15314`).
 - :meth:`Series.resample` and :meth:`DataFrame.resample` have gained the :meth:`Resampler.quantile` (:issue:`15023`).
 - :meth:`Index.to_frame` now supports overriding column name(s) (:issue:`22580`).
-
+- New attribute :attr:`__git_version__` will return git commit sha of current build (:issue:`21295` ).
```

Comment:
small nitpick: extra space before the closing parenthesis in <code>(:issue:\`21295\` )</code>

Reply:
updated